recipes/hasasia/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "hasasia" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87ab202bb6c07b24922169fc12115f9fe074c9d6ae33df08b91cb3c4ca48436f
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
For noarch:
```suggestion
    - python >=3.5
```

Reply:
@dopplershift Thanks for the suggestions!